setup.cfg

```diff
@@ -55,3 +64,6 @@ input_file = stestr/locale/stestr.pot
 keywords = _ gettext ngettext l_ lazy_gettext
 mapping_file = babel.cfg
 output_file = stestr/locale/stestr.pot
+
+[wheel]
+universal = 1
```

Comment:
Why is this needed? (I don't actually know what this does)

Reply:
oh, I just copy & paste from tempest code.. I should have removed this. I'll remove this.